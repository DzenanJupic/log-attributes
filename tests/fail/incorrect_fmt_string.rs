#[log_attributes::log(info, "} oh no {")]
fn incorrect_fmt_string() {}

fn main() {}
