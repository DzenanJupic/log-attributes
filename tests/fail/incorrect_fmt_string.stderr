error: invalid format string: unmatched `}` found
 --> tests/fail/incorrect_fmt_string.rs:1:30
  |
1 | #[log_attributes::log(info, "} oh no {")]
  |                              ^ unmatched `}` in format string
  |
  = note: if you intended to print `}`, you can escape it using `}}`
